extensions/schema-compare/src/test/testUtils.ts

```diff
@@ -112,11 +113,11 @@ export function setDacpacEndpointInfo(path: string): mssql.SchemaCompareEndpoint
 export function setDatabaseEndpointInfo(): mssql.SchemaCompareEndpointInfo {
 	let endpointInfo: mssql.SchemaCompareEndpointInfo;
 	let dbName = 'My Database';
-	let serverName = 'My Server';
+	let serverDisplayName = 'My Connection';
```

Comment:
While you're here, could you make these consts?

Reply:
done